lt/cmd/speech/main.go

```diff
@@ -18,7 +18,7 @@ import (
 )
 
 const (
-	duration = 10 * time.Minute
+	duration = 11 * time.Minute
```

Comment:
What's the context on this change? :p

Reply:
Because we're reusing the connect, it disconnects the scripted user after duration, even if the script isn't done. Since the script is around 10 mins, I'm just bumping it above that. But it makes more sense to turn this into a flag.
I should create a cfg and pass that instead of using vars. But I'll leave that for later if we need it.

Reply:
@cpoile I am not 100% convinced by this. I think it would be fair and likely simpler to ignore any duration if we are working with a script. The length of the script will determine how long it'll run. Thoughts?

Reply:
Sure, sounds good. I'd prefer to turn off the duration for scripts, but I didn't want to add a complicated conditional duration ticker in the connect function. So I opted to just set a long max duration for scripts (24 hours).

Reply:
Makes sense :+1: